src/IssueWanted/Db/Repo.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE DataKinds        #-}
+{-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE TypeOperators    #-}
+
+module IssueWanted.Db.Repo where
+
+import Squeal.PostgreSQL ((:::), NullityType (..), PGType (..))
+import Squeal.PostgreSQL.Schema (ColumnType (..))
+
+import qualified GHC.Generics as GHC
+
+
+type Schema =
+    '[ "repos"  ::: -- all Haskell repos
+        '[ "repoId"       ::: 'Required ('NotNull 'PGint4)
+         , "repoName"     ::: 'Required ('NotNull 'PGtext)
+         , "repoUrl"      ::: 'Required ('NotNull 'PGtext)
+         , "repoOwner"    ::: 'Required ('NotNull 'PGtext)
+         , "repoIssueHW"  ::: 'Required ('NotNull 'PGbool) -- Repo has issues labeled "HelpWanted"
+         , "repoIssueGFI" ::: 'Required ('NotNull 'PGbool) -- Repo has issues labeled "Good First Issue"
+        ]
+    , "repoIssueGFI" ::: -- all Haskell issues labeled "Good First Issue"
+        '[ "gfiTitle"  ::: 'Optional ('NotNull ' PGtext)
+         , "gfiUrl"    ::: 'Optional ('NotNull 'PGtext)
+         , "gfiRepoId" ::: 'Required ('NotNull 'PGint4)
+        ]
+
+    , "repoIssueHW" ::: -- all Haskell issues labeled "Help Wanted"
+        '[ "hwTitle"  ::: 'Optional ('NotNull 'PGtext)
+         , "hwUrl"    ::: 'Optional ('NotNull 'PGtext)
+         , "hwRepoId" ::: 'Required ('NotNull 'PGint4)
+        ]
```

Comment:
Looks like I need to read about `squeal` as well :joy: Looks legit at first glance. But need to double check. Also, I saw that `squeal` supports pretty-printing output of SQL query for creating table. Maybe it's a good idea to add it as a comment to this schema type? :thinking:

Reply:
> I saw that squeal supports pretty-printing output of SQL query for creating table. 

Could you share the link?

Reply:
This section of tutorial contains it:

* https://github.com/morphismtech/squeal#usage

Reply:
Sorry, I couldn`t find pretty section :(

Reply:
Well, it might not be pretty, just Haskell translated to SQL. Goes after:

> We can easily see the generated SQL is unsurprising looking.